app/store/utils/api.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
```

Comment:
- [ ] Can't we define this in `/app/setup/plugins/axios`? Please call me so we can discuss.